incubator/java-microprofile/templates/default/.appsody-config.yaml

```diff
@@ -1 +1 @@
-stack: appsody/java-microprofile:0.2
+stack: appsody/java-microprofile:0.2.5
```

Comment:
Let's keep this at **0.2** to pick up the latest

Reply:
good catch.. fixed